echopype/convert/ek80.py

```diff
@@ -545,6 +546,31 @@ def _export_zarr(self, save_settings, file_idx=0):
         else:
             self._set_groups(raw_file, out_file, save_settings=save_settings)
 
+    def _copy_group(self, src_file, trg_file):
+        # Utility function for copying the filter coefficients from one file into another
+        src = netCDF4.Dataset(src_file)
+        trg = netCDF4.Dataset(trg_file, mode='a')
+        ds_vend = src.groups['Vendor']
+        vdr = trg.createGroup('Vendor')
+        complex64 = np.dtype([("real", np.float32), ("imag", np.float32)])
+        complex64_t = vdr.createCompoundType(complex64, "complex64")
+
+        # set decimation values
+        vdr.setncatts({a: ds_vend.getncattr(a) for a in ds_vend.ncattrs()})
+
+        # Create the dimensions of the file
+        for k, v in ds_vend.dimensions.items():
+            vdr.createDimension(k, len(v) if not v.isunlimited() else None)
+
+        # Create the variables in the file
+        for k, v in ds_vend.variables.items():
+            data = np.empty(len(v), complex64)
+            var = vdr.createVariable(k, complex64_t, v.dimensions)
+            var[:] = data
+
+        src.close()
+        trg.close()
+
     def _combine_files(self):
```

Comment:
I think the name is different from the parent class, so the method is not inherited correctly?